### PR TITLE
Disk provisioning types vmware

### PR DESCRIPTION
**Pull request by Spaceman1984**
Changes related to VMware disk provisioning types.

### CODE REVIEW

File: source/adminguide/service_offerings.rst [hunk 1]
```diff
@@ -187,8 +187,17 @@ To create a new compute offering:
       system VM is running. Shared allocates from storage accessible via
       NFS.
 
-   -  **Provisioning type**: The type of disk that should be allocated. 
-      Local
+   -  **Provisioning type**: The type of disk that should be allocated.
+      Valid values are thin, sparse, fat. When using the VMWare hypervisor,
+      these values map as follows:
```
Comment (sureshanaparti) [inline source/adminguide/service_offerings.rst:192]:
```suggestion
      these values are mapped to the following vSphere disk provisioning types:

-  **thin** - **Thin Provision**
- **sparse** - **Thick Provision Lazy Zeroed**
- **fat** -  **Thick Provision Eager Zeroed**
```

File: source/adminguide/service_offerings.rst [hunk 2]
```diff
@@ -187,8 +187,17 @@ To create a new compute offering:
...
+
+      -  **Thin** - **Thin**.
+      -  **Sparse** - **Thick provisioned lazily zeroed**.
+      -  **Fat** -  **Thick provisioned eagerly zeroed**.
+      
+      The disk provisioning type strictness on VMWare iscontrolled with the zone level setting - **disk.provisioning.type.strictness**
```
Comment (sureshanaparti) [inline source/adminguide/service_offerings.rst:198]:
```suggestion
      The disk provisioning type strictness on VMWare is controlled with the zone level setting - **disk.provisioning.type.strictness**
```

File: source/adminguide/service_offerings.rst [hunk 3]
```diff
@@ -383,6 +392,18 @@ To create a new disk offering:
    -  **Disk Size**: Appears only if Custom Disk Size is not selected.
       Define the volume size in GB (2^30 1GB = 1,073,741,824 Bytes).
 
+   -  **Provisioning type**: The type of disk that should be allocated.
+      Valid values are thin, sparse, fat. When using the VMWare hypervisor,
+      these values map as follows:
```
Comment (sureshanaparti) [inline source/adminguide/service_offerings.rst:397]: same suggestion as above

File: source/adminguide/service_offerings.rst [hunk 4]
```diff
@@ -187,8 +187,17 @@ To create a new compute offering:
...
+
+      -  **Thin** - **Thin**.
+      -  **Sparse** - **Thick provisioned lazily zeroed**.
+      -  **Fat** -  **Thick provisioned eagerly zeroed**.
+      
+      The disk provisioning type strictness on VMWare iscontrolled with the zone level setting - **disk.provisioning.type.strictness**
+
```
Comment (sureshanaparti) [inline source/adminguide/service_offerings.rst:199]: white space not required, as this stmt is related to the setting.

File: source/adminguide/service_offerings.rst [hunk 5]
```diff
@@ -383,6 +392,18 @@ To create a new disk offering:
...
 
+   -  **Provisioning type**: The type of disk that should be allocated.
+      Valid values are thin, sparse, fat. When using the VMWare hypervisor,
+      these values map as follows:
+
+      -  **Thin** - **Thin**.
+      -  **Sparse** - **Thick provisioned lazily zeroed**.
+      -  **Fat** -  **Thick provisioned eagerly zeroed**.
+      
+      The disk provisioning type strictness on VMWare iscontrolled with the zone level setting - **disk.provisioning.type.strictness**
```
Comment (sureshanaparti) [inline source/adminguide/service_offerings.rst:403]: check if this is redundant

File: source/quickinstallationguide/qig.rst [hunk 6]
```diff
@@ -433,6 +433,13 @@ Install Python MySQL connector from the MySQL community repository (which we've
 Please note that the previously required ``mysql-connector-java`` library is now bundled with CloudStack
 Management server and is no more required to be installed separately.
```
Comment (sureshanaparti) [inline source/quickinstallationguide/qig.rst:435]: Seems the changes here, are not related to disk provisioning types. Please check.

Comment (Spaceman1984) [reply source/quickinstallationguide/qig.rst:435]: This was a conflict that I had to resolve

File: source/adminguide/service_offerings.rst (hunk 3 above)
Comment (Spaceman1984) [reply source/adminguide/service_offerings.rst]: Done

File: source/adminguide/service_offerings.rst (hunk 4 above)
Comment (Spaceman1984) [reply source/adminguide/service_offerings.rst:197]: Done

File: source/adminguide/service_offerings.rst (hunk 5 above)
Comment (Spaceman1984) [reply source/adminguide/service_offerings.rst]: Done

File: source/quickinstallationguide/qig.rst (hunk 6 above)
Comment (sureshanaparti) [reply source/quickinstallationguide/qig.rst:435]: can you rebase and resolve properly. this changes shouldn't come up as changes in this PR.

**Review by sureshanaparti — APPROVED** [review]

**Comment by DaanHoogland** [comment]
@rhtyd I think the code PR is for 4.16. should this doc-change be marked for 4.15?

**Comment by rhtyd** [comment]
:facepalm: nope you're right @DaanHoogland @Spaceman1984 can you open a PR for master branch, I'll revert the change on 4.15 branch.

**Comment by Spaceman1984** [comment]
Created a new pull request #217

**Comment by DaanHoogland** [comment]
revert as PR #218 not sure if we need reviews there @rhtyd  so merge at will.

**Comment by rhtyd** [comment]
Thanks @DaanHoogland

**Pull request merged**